tingbot/graphics.py

```diff
@@ -163,8 +163,18 @@ def surface(self):
     def size(self):
         return self.surface.get_size()
 
+    def _fill(self, color, rect = None):
+        if len(color)<=3:
+            self.surface.fill(color,rect)
+        elif len(color)>=4:
+            if rect is None:
+                rect = pygame.Rect((0,0),self.size)
+            tmp_surface = pygame.Surface(rect.size,pygame.SRCALPHA)
```

Comment:
`rect` is a tuple if we come from screen.rectangle, so need to make that into a pygame rect here

            if rect is None:
                rect = pygame.Rect((0,0),self.size)
    +       elif isinstance(rect, (tuple, list)):
    +           rect = pygame.Rect(*rect)
            tmp_surface = pygame.Surface(rect.size,pygame.SRCALPHA)
            tmp_surface.fill(color)